tracing-test/Cargo.toml

```diff
@@ -13,12 +13,12 @@ categories = ["development-tools::testing"]
 [dependencies]
 lazy_static = "1.4"
 tracing-core = "0.1"
-tracing-subscriber = "0.2"
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
```

Comment:
The `env-filter` feature is already enabled by default, but it doesn't hurt to explicitly list it!

Reply:
It's actually no longer enabled by default for tracing_subscriber 0.3 ;)
ref: https://docs.rs/crate/tracing-subscriber/0.3.2/features

Reply:
Oops, thanks for the correction, I accidentally looked at the outdated docs.rs page for 0.2 🙂